index.html

```diff
@@ -15,157 +15,200 @@
+                    <h1 class="project-title-heading text-desktop">Uber Navigation</h1>
+
+                    <article class="frame-2">
+                        <h2 class="client-heading text-mobile">CANOPY</h2>
+                        <h2 class="client-heading text-desktop">Uber</h2>
+                        <img src="./assets/icons/Counter.png" alt="Counter">
+                        <h2 class="role text-mobile">Back End Dev</h2>
+                        <h2 class="role text-desktop">Lead Developer</h2>
+                        <img src="./assets/icons/Counter.png" alt="Counter">
+                        <h2 class="year text-mobile">2015</h2>
+                        <h2 class="year text-desktop">2018</h2>
+                    </article>
+
+                    <p class="project-info-p text-mobile">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+
+                    <p class="project-info-p text-desktop">
+                    A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+                    </p>
+
+                    <ul class="logo-tags">
+                        <li><a href="#"><img src="./assets/icons/Htmllogo.png" alt="Htmllogo"></a></li>
+                        <li><a href="#"><img src="./assets/images/ruby-n-rails-tag.png" alt="Ruby on rails" class="portfolio-img-1-desktop"></a></li>
+                        <li><a href="#"><img src="./assets/icons/csslogo.png" alt="csslogo"></a></li>
+                        <li><a href="#"><img src="./assets/icons/jslogo.png" alt="jslogo"></a></li>
+                    </ul>
+                    <div class="button-container">
+                        <button class="button-see-project" type="button">
+                            See project
+                        </button>
+                    </div>
                 </article>
-
-                <p class="project-info-p">
-                    A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </p>
-
-                <ul class="logo-tags">
-                    <li><a href="#"><img src="./assets/icons/Htmllogo.png" alt="Htmllogo"></a></li>
-                    <li><a href="#"><img src="./assets/icons/csslogo.png" alt="csslogo"></a></li>
-                    <li><a href="#"><img src="./assets/icons/jslogo.png" alt="jslogo"></a></li>
-                </ul>
-                <div class="button-container">
-                    <button class="button-see-project" type="button">
-                        See project
-                    </button>
-                </div>
+                <img src="./assets/images/uber-nav-desktop.svg" alt="Snapshot Portfolio" class="portfolio-img-1-desktop">
             </article>
         </section>
         <section class="about-myself-section">
```

Comment:
Please, update the design of this section to stick to Figma,
In your design, each element has 50% of the width
In Figma, the 1st part takes around 40% and the 2nd around 60%
<img width="655" alt="image" src="https://user-images.githubusercontent.com/7243132/171746578-37e4362a-b38f-4f60-b6f7-4446676f4d9a.png">